api/models/fixtures/web_external_internal.json

```diff
@@ -165,14 +178,32 @@
         ],
         "Listeners": [
           {
-            "InstancePort": {
-              "Ref": "WebPort80Host"
-            },
-            "InstanceProtocol": "TCP",
-            "LoadBalancerPort": {
-              "Ref": "WebPort80Balancer"
-            },
-            "Protocol": "TCP"
+            "Fn::If": [
+              "BlankWebPort80Certificate",
+              {
+                "InstancePort": {
+                  "Ref": "WebPort80Host"
+                },
+                "InstanceProtocol": "TCP",
+                "LoadBalancerPort": {
+                  "Ref": "WebPort80Balancer"
+                },
+                "Protocol": "TCP"
+              },
+              {
+                "InstancePort": {
+                  "Ref": "WebPort80Host"
+                },
+                "InstanceProtocol": "TCP",
```

Comment:
Rack is managed by kernel.json.

Grid has a manually configured listener per the current [SSL setup instructions](http://convox.com/docs/custom-certificates/). When this patch lands add one more line to the grid manifest, deploy, and re-upload a cert.

```diff
$ git diff docker-compose.yml
diff --git a/docker-compose.yml b/docker-compose.yml
index 28948e2..f5e20b2 100644
--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -14,6 +14,7 @@ web:
     - GRID_HOST
   ports:
     - 80:5000
+    - 443:5000
     - 4443
   command: bin/web
   volumes:
```